src/legacy/core_plugins/metrics/public/components/vis_editor.js

```diff
@@ -134,12 +113,11 @@ class VisEditor extends Component {
       if (!this.props.vis.params || !this.props.visData) {
         return null;
       }
-      const reversed = this.state.reversed;
       return (
         <I18nProvider>
           <Visualization
             dateFormat={this.props.config.get('dateFormat')}
-            reversed={reversed}
+            reversed={false}
```

Comment:
Will base this value on the dark mode uiSetting in https://github.com/elastic/kibana/pull/28445

Reply:
https://github.com/elastic/kibana/pull/28445/files#diff-1cbce47c73f79af204a21f49b168b394R122